ospatch/updates_windows.go

```diff
@@ -166,60 +166,61 @@ func (r *patchRun) installWUAUpdates() error {
 	}
 	defer session.Release()
 
-	r.debugf("Searching for available WUA updates.")
-	updts, err := packages.GetWUAUpdateCollection(session, "IsInstalled=0")
-	if err != nil {
-		return fmt.Errorf("GetWUAUpdateCollection error: %v", err)
-	}
-
-	countRaw, err := updts.GetProperty("Count")
-	if err != nil {
-		return err
-	}
-	count, _ := countRaw.Value().(int32)
+	// We keep searching for and installing updates until the count == 0 or there is an error.
+	for {
```

Comment:
Should we put a max updates here just to be safe? It could even be a high number like 500 or 1000. Just to make sure that we never enter a weird state where we detect an update over and over again (maybe because it fails to install in some weird way).